catalog/config.json.example

```diff
@@ -0,0 +1,29 @@
+{
+  "alwaysRequiresAuth": false,
+  "sentryDSN": "<DSN here>",
+  "apiGatewayUrl": "https://ozj6r9bglc.execute-api.us-east-1.amazonaws.com/prod",
```

Comment:
please do the same as with sentry, replace with `<apiGateway url here>` or similar